src/test/java/at/aau/serg/websocketdemoserver/WebSocketBrokerIntegrationTest.java

```diff
@@ -193,6 +196,64 @@ public void testWebSocket_GetPlayerTurnUpdate() throws Exception {
 
         //TODO: Add tests if the message for the active player came
     }
+    /*
+    @Test
+    void testPlayCardEndpoint() throws Exception {
+        String userID = "123";
```

Comment:
Setup, damit man überhaupt eine Karte spielen kann ist schon realtiv groß hier.
Vielleicht sollten wir eine initPlayableLobby (oder ähnlich) Methode machen. Damit wir hier wirklich nur auf das Karte spielen testen müssen

Reply:
Hab jetzt eine Art Lobby Setup geschrieben, hab irgendwie ein selstames Issue wo ich einen String in "messages" habe der da nicht sein sollte, muss ich mir weiter anschauen